### PR TITLE
fix broken specs, warnings and deprecations

### PR DESCRIPTION
**Pull request by yunixon**
https://trello.com/c/4f4ZEGIq/439-fix-failed-specs-deprecations

### CODE REVIEW

File: spec/requests/api/v1/bank_accounts/index_spec.rb [hunk 1]
```diff
@@ -18,19 +18,23 @@
     it 'returns bank_accounts' do
       expect(response).to be_success
 
-      expect(json[0]).to include(
-        'id'       => bank_account2.id,
-        'name'     => bank_account2.name,
-        'currency' => bank_account2.currency,
-        'balance'  => bank_account2.balance.as_json,
-        'residue'  => bank_account2.residue.as_json,
+      expect(json).to include(
```
Comment (shir) [inline spec/requests/api/v1/bank_accounts/index_spec.rb]: @yunixon not sure why do you need this fix. `json` should have array of bank_accounts. Why you change to compare `json` instead `json[0]`?

File: spec/support/capybara.rb [hunk 2]
```diff
@@ -4,9 +4,13 @@
...
 
-# Capybara::Webkit.configure do |config|
-#   config.skip_image_loading
-# end
+Capybara::Webkit.configure do |config|
+  # Silently return an empty 200 response for any requests to unknown URLs.
+  config.block_unknown_urls
+
+  # Timeout if requests take longer than 20 seconds
+  config.timeout = 20
```
Comment (shir) [inline spec/support/capybara.rb]: @yunixon this timeout looks to long for tests. Change it to 5 seconds.

Comment (yunixon) [reply spec/support/capybara.rb]: @shir with 5 seconds some specs randomly failed

Comment (shir) [reply spec/support/capybara.rb]: @yunixon it's the issue you should fix. No requests should take more then 5 seconds.

**Pull request merged**